src/NuGet.Indexing/PerfEventTracker.cs

```diff
@@ -7,6 +7,7 @@
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
+using NuGetGallery;
```

Comment:
I think this was to allow access to the disposableAction below.
Might it make more sense to just copy the disposableAction into Nuget.Indexing?
Also, i'm not 100% sure how the reference to disposableAction was being resolved before, as the current one is referecning NuGetGallery.DisposableAction.

Maybe it had to do with the reference to NuGet.Core that i think i dropped.

In any case, what do you think we should do here?